llvm/test/DebugInfo/symbolize-big-archive-xcoff.test

```diff
@@ -0,0 +1,27 @@
+// Test big archive recognition and error handling in llvm-symbolizer
+// REQUIRES: system-aix, target={{.*}}-aix{{.*}}
+
+// Generate object files
+// RUN: yaml2obj -o %t-32.o %S/Inputs/big-archive-32.yaml
+// RUN: yaml2obj -o %t-64.o %S/Inputs/big-archive-64.yaml
+
+// Create archive with same-named members using different modes
+// RUN: rm -f %t.a
+// RUN: cp %t-32.o %t.o && llvm-ar -X32 crv %t.a %t.o
+// RUN: cp %t-64.o %t.o && llvm-ar -X64 qv %t.a %t.o
```

Comment:
On AIX, the second `llvm-ar` invocation should still use `r` instead of `q` to emphasize that adding a 64-bit member does not replace the 32-bit member.

On non-AIX platforms, omit the `-X32` and the `-X64`.